### PR TITLE
[IDB IGNORE] [MDB IGNORE] More uniform conversions.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
DNM, opening for CI. Depends on #3948.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
This one is a bit halfassed but it does the job of killing /under. I'll do another final PR after this to go through /costume and convert what can be converted.

File: mods/content/government/away_sites/icarus/icarus.dm [hunk 1]
```diff
@@ -164,11 +164,6 @@
 	icon = 'mods/content/government/away_sites/icarus/icarus_sprites.dmi'
 	icon_state = "solgovseal"
 
-/obj/item/clothing/under/ec_uniform
-	name = "expeditionary uniform"
-	desc = "An older model of the utility uniform of the SCG Expeditionary Corps. It has a patch on the left sleeve signifying the wearer served on the SEV Icarus."
-	icon = 'icons/clothing/under/uniform_ec.dmi'
```
Comment (MistakeNot4892) [inline mods/content/government/away_sites/icarus/icarus.dm:170]: This icon turned out to not have anything in it, for the record.

**Review by out-of-phaze — DISMISSED** [review]
doesn't look like this has a map migration, and it needs one

**Comment by MistakeNot4892** [comment]
> doesn't look like this has a map migration, and it needs one

Done.

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**